packages/css/src/popover.css

```diff
@@ -55,3 +55,13 @@
     --dsc-popover-background: var(--ds-color-neutral-background-default);
   }
 }
+
+.ds-popover--trigger-inline {
```

Comment:
Maybe this could be data attribute to match the rest of our css where we use that for our modifiers?

Reply:
I think it would be nice if this specifies that it is for popover 🤔

Reply:
Godt poeng! hva med:
```
[data-popover="inline"] {
  ...
}
```